samples/Readme.md

```diff
@@ -0,0 +1,18 @@
+# Maestro samples
+
+`maestro download-samples` provides a set of flows and apps so that users can quickly set up a maestro test, without having to create an app.
+
+download-samples downloads these files and apps from storage.googleapis.com.
+
+# Update the samples
+
+Although the samples are checked in, updating them requires a few manual steps:
+
+* Change the samples in this directory and merge these changes
+* Run `maestro download-samples`
+* Copy *.yaml to the samples directory created by download-samples
+* Run `zip -r samples.zip samples -x ".*" -x "__MACOSX"`
+* Open https://console.cloud.google.com/storage/browser/mobile.dev/samples
```

Comment:
The link https://storage.googleapis.com/mobile.dev/samples/samples.zip is already in the repo, so that maestro can download the zip file. This link does not reveal that much extra.

Reply:
Ahh got it. LGTM 💯